README.md

```diff
@@ -30,6 +30,23 @@ Given **OpenVINO** models and input images, **OpenVINO XAI** generates **salienc
 which highlights regions of the interest in the inputs from the models' perspective
 to help users understand the reason why the complex AI models output such responses.
 
+Using this package, you can augment the model analysis & explanation feature
```

Comment:
It's just a suggestion, but what do you think about adding the actual "Why OpenVINO-XAI" heading here?

Reply:
I don't think the "a few lines of code" is that much attractive for the heading.
I've just put a small "why" after the small "what".

Reply:
We might probably need "Why OpenVINO-XAI" section somewhere in the readme.

But this new lines better be named "Where and how OV XAI". Although can be skipped from my perspective